### ISSUE TITLE
Postgres table are not listed in alphabetical order

### ISSUE THREAD

**Issue by frankbreetz** [tconbeer/harlequin#390]
**Describe the bug**
The Tables listed are not in Alphabetical Order, this makes it difficult to find what table you are looking for

**To Reproduce**
 I am not sure of the order of the tables, seems random to me. 
So make a bunch of tables in postgres and look at the order in harlequin

**Expected behavior**

The tables should be listed in Alphabetical order or there should be a config option to make this happen

**Actual behavior**
Tables listed in non-alphbetical order:

![harlequin tables](https://github.com/tconbeer/harlequin/assets/24467723/524972b8-28ae-4df7-be9a-471f47fcc57e)

**Additional context**

harlequin, version 1.8.0

Installed Adapters:
  - postgres, version 0.2.1
  - duckdb, version 1.8.0
  - sqlite, version 1.8.0

What database adapter are you using with Harlequin?  Postgres

Can you tell us more about your system?
- Shell: omgzsh
- Terminal: Wezterm
- OS of the shell: WSL2/Ubuntu 22.04
- OS of the terminal (if different from the shell): Windows 10

**Comment by tconbeer** [comment]
agreed. this is an easy fix if you want to contribute to the harlequin-postgres project?
